### PR TITLE
Add administrative

### PR DESCRIPTION
**Pull request by nitrocode**
## what
* Add administrative

## why
* Be able to create administrative stacks by updating a boolean
* This would be necessary if we needed to create an service specific admin stack for its preview environment
* This also opens up managing administrative stacks by using the `modules/stack` module directly

## references
N/A

### CODE REVIEW

File: modules/stack/main.tf [hunk 1]
```diff
@@ -6,19 +6,18 @@ resource "spacelift_stack" "default" {
   count = var.enabled ? 1 : 0
 
   name                 = var.stack_name
-  administrative       = false
+  administrative       = var.administrative
```
Comment (aknysh) [inline modules/stack/main.tf:9]: looks like `administrative` is not used in the stack submodule
@nitrocode

**Review by aknysh — COMMENTED** [review]
please see comments

File: modules/stack/variables.tf [hunk 2]
```diff
@@ -178,3 +178,9 @@ variable "stack_destructor_enabled" {
   description = "Flag to enable/disable the stack destructor to destroy the resources of the stack before deleting the stack itself"
   default     = false
 }
+
+variable "administrative" {
+  type        = bool
+  description = "Whether this stack can manage others"
```
Comment (aknysh) [inline modules/stack/variables.tf:184]:
```suggestion
  description = "Whether this stack can manage other stacks"
```

File: variables.tf [hunk 3]
```diff
@@ -241,3 +241,9 @@ variable "stack_destructor_enabled" {
   description = "Flag to enable/disable the stack destructor to destroy the resources of a stack before deleting the stack itself"
   default     = false
 }
+
+variable "administrative" {
+  type        = bool
+  description = "Whether this stack can manage others"
```
Comment (aknysh) [inline variables.tf:247]:
```suggestion
  description = "Whether this stack can manage other stacks"
```

File: modules/stack/variables.tf (hunk 2 above)
Comment (nitrocode) [reply modules/stack/variables.tf:184]: This was a copy from the spacelift stack resource but it doesn't mean we have to be consistent with poor wording 😉

**Review by aknysh — APPROVED** [review]

**Pull request merged**